TelCo.ColorCoder/NumberPair.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+   
+    class NumberPair
```

Comment:
`NumberPair` indicates a pair of numbers. But this class deals with a pair of colors. What would be a better name for this class? What do the consumers of this class expect from it?

Reply:
I have worked on it and updated the class names according to what the function in the class returns. Is that a correct way? Kindly let me know.